client/lib/post-normalizer/index.js

```diff
@@ -509,6 +511,7 @@ normalizePost.content = {
 					// fun fact: removing the node from the DOM will not prevent it from loading. You actually have to
 					// change out the src to change what loads. The following is a 1x1 transparent gif as a data URL
 					image.setAttribute( 'src', 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7' );
+					image.removeAttribute( 'srcset' );
```

Comment:
you know... i wonder if what i was seeing was a srcset being present along with a src and that tricked me into thinking it was the src loading the image..

assume you're referring to the comment above